projects/laji/src/app/+observation/form/observation-form.component.scss

```diff
@@ -25,13 +25,13 @@ laji-select, laji-metadata-select {
 hr {
   margin: 4px 0;
 }
-#logCoordinateAccuracyMax:focus {
+#coordSeparator {
```

Comment:
ids are exclusive (you shouldn't have two elements with the same id on the same page). Use a class instead